packages/manager/src/features/Users/UserRow.tsx

```diff
@@ -36,6 +40,11 @@ export const UserRow = ({ onDelete, user }: Props) => {
         <TableCell>{user.email}</TableCell>
       </Hidden>
       <TableCell>{user.restricted ? 'Limited' : 'Full'}</TableCell>
+      {flags.parentChildAccountAccess && (
+        <TableCell>
+          {grants?.global.child_account_access ? 'Enabled' : 'Disabled'}
```

Comment:
```suggestion
          {grants?.global?.child_account_access ? 'Enabled' : 'Disabled'}
```
With the feature flag on and mocks OFF, we ended up with an Oh Snap crash on this page:
![image](https://github.com/linode/manager/assets/114685994/7cafb810-8f59-4639-82da-dd09e8c0b1c0)